app/apollo/resolvers/channel.js

```diff
@@ -302,6 +324,42 @@ const channelResolvers = {
         throw err;
       }
     },
+    removeChannelVersion: async (parent, { org_id, uuid }, context)=>{
```

Comment:
we need to support S3. see here: https://github.com/razee-io/Razeedash-api/blob/master/app/apollo/resolvers/channel.js#L212 for details.